app/human-detector-app/classes/User.tsx

```diff
@@ -35,4 +36,17 @@ export default class User {
     this.groupList.splice(groupIndex, 1);
     return groupRemoved;
   }
+
+  // eslint-disable-next-line class-methods-use-this
+  getAllNotifications(allNotifications: Notification[], groups: Group[]): Notification[] {
+    for (let i = 0; i < groups.length; i += 1) {
```

Comment:
nit: I know you kind of implemented it before when troubleshooting, but would you be able to use the `map` function instead of for loops?  It would clean it up a bit